api/src/main/java/org/openmrs/PatientIdentifierType.java

```diff
@@ -32,7 +33,8 @@
 @Table(name = "patient_identifier_type")
 @Audited
 @AttributeOverrides({
-	@AttributeOverride(name = "name", column = @Column(name = "name", nullable = false, length = 50))
+	@AttributeOverride(name = "name", column = @Column(name = "name", nullable = false, length = 50)),
+	@AttributeOverride(name = "description", column = @Column(name = "description", columnDefinition = "text"))
```

Comment:
Why should a patient identifier type description have a column definition of text?